setup.cfg

```diff
@@ -3,3 +3,7 @@ universal: 1
 
 [tool:pytest]
 testpaths: affine/tests
+
+[pydocstyle]
+select: D1
+add-ignore: D105
```

Comment:
This package has a lot of magic methods and they don't all need docstrings when they follow the Python object semantics.